### ISSUE TITLE
Where's Paradise Station?

### ISSUE THREAD

**Issue by BlitzkriegEd31** [ParadiseSS13/Paradise#7211]
**Problem Description**:
It wont show paradise station on the SS13 game list when I try to play it.

**What did you expect to happen**: 
I expected a paradise link to pop up under the SS13 servers.

**What happened instead**:
Paradise station won't appear as a server.

**Why is this bad/What are the consequences:**
I simply can't play the game anymore.

**Steps to reproduce the problem**:
I'm a usual paradise station player and I usually log in once or twice a day and it rarely shows the link for the server.

**Possibly related stuff (which gamemode was it? What were you doing at the time? Was
anything else out of the ordinary happening?)**: Nothing weird was happening or happened leading up to this problem.

**Comment by SamHPurp** [comment]
Hey @BlitzkriegEd31 , the server has begun de-listing itself during high-pop to stop us from overflowing with people.

The best way to get to the server is to add it to favourites at nanotrasen.se:6666

Alternatively https://nanotrasen.se/forum/ and click Join Server.

**Comment by FalseIncarnate** [comment]
This is not a code issue and thus does not belong on the github.

However, we have recently implemented a system that delists the server when we reach certain population criteria, so you likely are looking during such times.

I recommend bookmarking the server in BYOND to ensure you always have a link right from the initial menu in BYOND, or (as Sam suggested) check our forums in case we have actually gone down (there'd be an announcement if we did) and to access our quick-join link there.
